community/kernel/src/test/java/org/neo4j/kernel/impl/storemigration/MigrationTestUtils.java

```diff
@@ -148,17 +148,24 @@ public static void prepareSampleLegacyDatabase( String version, FileSystemAbstra
 
     public static File findFormatStoreDirectoryForVersion( String version, File targetDir ) throws IOException
     {
-        switch ( version )
+        if ( version.equals( StandardV2_3.STORE_VERSION ) )
```

Comment:
If you will do the following in `MigrationTestUtils`
```java
public static File findFormatStoreDirectoryForVersion( String version, File targetDir ) throws IOException
    {
        StoreVersion storeVersion = StoreVersion.valueOfByVersionString( version );
        switch ( storeVersion )
        {
        case STANDARD_V2_0:
            return find20FormatStoreDirectory( targetDir );
        case STANDARD_V2_1:
            return find21FormatStoreDirectory( targetDir );
        case STANDARD_V2_2:
            return find22FormatStoreDirectory( targetDir );
        case STANDARD_V2_3:
            return find23FormatStoreDirectory( targetDir );
        default:
            throw new IllegalArgumentException( "Unsupported version " + storeVersion );
        }
    }
```
and following in `StoreVersion`:
```java
    public static StoreVersion valueOfByVersionString(String versionString)
    {
        for ( StoreVersion storeVersion : ALL_STORE_VERSIONS )
        {
            if (storeVersion.string.equals( versionString ))
            {
                return storeVersion;
            }
        }
        throw new IllegalArgumentException("Unknown store version string: " + versionString);
    }
```
looks better right?